aspy/refactor_imports/classify.py

```diff
@@ -39,12 +48,12 @@ def _is_a_local_path(potential_path: str) -> bool:
         abspath = os.path.abspath(module_path)
         realpath = os.path.realpath(module_path)
         return (
-            abspath.startswith(localpath) and
+            _has_path_prefix(abspath, prefix=localpath) and
             # It's possible (and surprisingly likely) that the consumer has a
             # virtualenv inside the project directory.  We'd like to still
             # consider things in the virtualenv as third party.
             os.sep not in abspath[len(localpath) + 1:] and
-            abspath == realpath and
+            os.path.samefile(abspath, realpath) and
```

Comment:
this doesn't work:

```pycon
>>> os.path.samefile('venv/bin/python', '/usr/bin/python3')
True
```

Reply:
d'uh... Changed to use `os.path.normcase` and string equality.